Source/Core/Core/PowerPC/JitArm64/Jit.cpp

```diff
@@ -344,8 +344,8 @@ void JitArm64::IntializeSpeculativeConstants()
         STR(IndexType::Unsigned, DISPATCHER_PC, PPC_REG, PPCSTATE_OFF(pc));
         MOVP2R(ARM64Reg::X8, &JitInterface::CompileExceptionCheck);
         MOVI2R(ARM64Reg::W0, static_cast<u32>(JitInterface::ExceptionType::SpeculativeConstants));
-        BLR(ARM64Reg::X8);
-        B(dispatcher_no_check);
+        MOVP2R(ARM64Reg::X30, dispatcher_no_check);
+        BR(ARM64Reg::X8);
```

Comment:
I'd find a comment saying something like "X30 is LR; perform a tail call to dispatcher_no_check after the exception check" useful here (and in the other change in this commit).